src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitor.java

```diff
@@ -65,6 +68,8 @@ private void watchForCertExpiryOnce() {
 
         for (CertificateGenerator cg = monitoredCertificateGenerators.peek(); cg != null;
              cg = monitoredCertificateGenerators.peek()) {
+            LOGGER.atInfo().kv("certificateSubject", cg.subject)
+                    .log("Checking certificate expiry, regenerate if necessary");
```

Comment:
This runs every 30 seconds. Log volume will be way too high

Reply:
removed.